src/main/java/net/atos/client/zgw/zrc/model/RolNietNatuurlijkPersoon.java

```diff
@@ -70,4 +70,15 @@ public int hashCode() {
         }
         return Objects.hash(getRoltype(), getBetrokkeneType(), null);
     }
+
+    @Override
+    protected int hashCodeBetrokkeneIdentificatie() {
+        if (getBetrokkeneIdentificatie().getAnnIdentificatie() != null) {
+            return Objects.hash(getBetrokkeneIdentificatie().getAnnIdentificatie());
+        }
+        if (getBetrokkeneIdentificatie().getInnNnpId() != null) {
+            return Objects.hash(getBetrokkeneIdentificatie().getInnNnpId());
+        }
+        return 1;
```

Comment:
Itt has no meaning really. I will change it to `0` because that is what was used before this PR. `Objects.hash(null)` was done before and this corresponds to `0`

From `Arrays.hashCode`:

>The value returned by this method is equal to the value that would be returned by Arrays.asList(a).hashCode(), unless a is null, in which case 0 is returned.

Reply:
I changed it to `0` and left it at that. Hope that's ok with you. ;-)